content/docs/intro/languages/dotnet.md

```diff
@@ -0,0 +1,61 @@
+---
+title: ".NET"
+menu:
+  intro:
+    parent: languages
+    weight: 2
+
+aliases: ["/docs/reference/dotnet/"]
+---
+
+Pulumi supports .NET programs running on .NET Core 3 and later. You can also use your favorite .NET tools such as editors, package managers, build systems, and test frameworks.
+
+## Getting Started
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Install .NET Core 3.0+](https://dotnet.microsoft.com/download)
```

Comment:
i thnk we should explicitly state you need the ".NET Core SDK 3.0" (the "sdk" bit is the important part).